introduction.md

```diff
@@ -1,2 +1,5 @@
 Hi!This is my first file in github!Be safe!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?